counterpartylib/lib/blocks.py

```diff
@@ -93,8 +93,8 @@ def parse_tx(db, tx):
         order.parse(db, tx, message)
     elif message_type_id == btcpay.ID:
         btcpay.parse(db, tx, message)
-    elif message_type_id == issuance.ID:
-        issuance.parse(db, tx, message)
+    elif message_type_id == issuance.ID or message_type_id == issuance.SUBASSET_ID:
```

Comment:
Looking at this, it's the only ```or``` logic in this group of else/if statements. It might be better as just another```elif```.

Really just a comment re: consistency.